### ISSUE TITLE
bluetooth keyboard support and english documentation

### ISSUE THREAD

**Issue by Dennisl123** [nirenr/jieshuo#26]
Hi, is there any bluetooth keyboard support for your screen reader?  Ideally I should be able to do everything with a bluetooth keyboard that you can do using the touch screen.  More over this would be clearly and properly documented in English with a list of the keyboard commands and what they do.  I couldn't find this If I Missed it please give me a link to the keyboard commands.  Thanks.
